src/renderer/fonts/font_options.rs

```diff
@@ -277,4 +300,43 @@ mod tests {
             FontHinting::Slight,
         );
     }
+
+    #[test]
+    fn test_parse_font_name_with_escapes() {
+        let without_escapes_or_specials_chars = parse_font_name("Fira Code Mono");
+        let without_escapes = parse_font_name("Fira_Code_Mono");
+        let with_escapes = parse_font_name("Fira\\_Code\\_Mono");
```

Comment:
For readability, you might want to use raw-strings:

```suggestion
        let with_escapes = parse_font_name(r"Fira\_Code\_Mono");
```